ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
 *	a collection of data reports and dashboards
 *	a news and updates area.
 
-As long as you're connected to the network - it's available to anyone who works for the Department of Education.
+As long as you're connected to the network - it's available to anyone who works for the Department of Education. 
 
-See these [Databricks support notebooks](https://adb-2220072380334347.7.azuredatabricks.net/browse/folders/4213691640199507?o=2220072380334347) for more detailed reference guides into how we have setup the service here at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. Please note that you will need a Databricks account to access these. 
+Databricks is currently in its onboarding phase, which is expected to continue until **???**. At the moment, teams can onboard voluntarily until **???**. Following this date, the remaining teams will have their data migrated for them. 
 
+See these [Databricks support notebooks](https://adb-2220072380334347.7.azuredatabricks.net/browse/folders/4213691640199507?o=2220072380334347) for more detailed reference guides into how the service has been set up at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. Please note that you will need a Databricks account to access these. 
 
----
+From the [ADA homepage](https://analytical-data-access.education.gov.uk/) you will be able to find data and access cloud analytical tools. This includes:
 
-# Plans for the future
+* Navigation to Data Discovery Platform to find information regarding metadata of the datasets, including how to request access and information about the quality of the data. The catalogue is supported by the Metadata Team and will eventually include all Department data, as well as tagging of datasets available through ADA.
 
----
+* Analytical workbench to analyse and model data. This is underpinned by Databricks, a cloud-based platform to load and store data; and perform analysis, modelling and machine learning of data; as well as output to various formats to share analysis. The ADA team are looking to extend integration to R Studio in the coming months.
```

Comment:
Is the last sentence the POSIT workbench stuff? If so I'd rephrase to 'In the coming months, the ADA team are looking to extend the platform to include POSIT workbench to allow R Studio to be used in the cloud.'

Reply:
I believe so, I will change it